Anatoly Shlom/flex/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Paris</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="css/flex.css">
+</head>
+<body>
+    <main class="widget">
+        <article class="widget-data">
+            <section class="current-date">
+                <div class="current-date__week-day">
+                    Friday
+                </div>
+                <div class="current-date__day">
+                    August, 23rd
+                </div>
+            </section>
+            <section class="weather-block">
+                <div class="weather-block__temperature">
+                    12&#176;
+                </div> 
+                <div class="weather-block__location">
+                    Paris, France
+                </div>
+                <div class="wind-block">
```

Comment:
Doesn't too matter, but why it's a block whether 'weather-block__location' is not?

Reply:
Hmm... looks fine as i see. But maybe I just not quite understood your point
![code](https://user-images.githubusercontent.com/4656313/61597799-47ca9a80-ac1d-11e9-8292-17d3eb27b0bf.png)

Reply:
That's okay, but looks like that `wind-block` is an element of `weather-block`